example/android/app/src/main/kotlin/com/jhomlala/better_player_example/BetterPlayerNotificationService.kt

```diff
@@ -37,10 +39,16 @@ class BetterPlayerNotificationService: Service() {
     // Observe update of notification action.
     private val notificationActionListObserver =
         Observer<List<NotificationCompat.Action>?> { actions ->
+            notificationBuilder?.clearActions()
             actions?.map { action ->
-                notificationBuilder?.clearActions()
                 notificationBuilder?.addAction(action)
             }
+            val mediaStyle =
+                androidx.media.app.NotificationCompat.MediaStyle().setMediaSession(mediaSessionToken)
+            if (actions.isNotEmpty()) {
+                mediaStyle.setShowActionsInCompactView(actions.size - 1)
```

Comment:
To hide button in notification, needed to update setShowActionsInCompactView based on count of button.
Will do same modification for BetterPlayerNotificationService in App side.